### PR TITLE
FILACMS-9: Add IsProtectedResource

### PR DESCRIPTION
**Pull request by kyoungportable**

### CODE REVIEW

File: src/Filament/Resources/AuthorResource.php [hunk 1]
```diff
@@ -31,12 +33,12 @@ public static function form(Form $form): Form
                 Fieldset::make()
                 ->schema([
                     TextInput::make('first_name')
-                        ->label(fn(Get $get) => $get('is_individual') ? 'First Name' : 'Organization Name')
+                        ->label(fn (Get $get) => $get('is_individual') ? 'First Name' : 'Organization Name')
```
Comment (Kashkin) [inline src/Filament/Resources/AuthorResource.php:36]: There is some linting happening that is distracting. Can we use prettier or editorconfig to document the desired standards?

**Review by Kashkin — COMMENTED** [review]
Requested changes are minor and not required. Branch tested locally in project with success.

File: src/Filament/Resources/AuthorResource.php (hunk 1 above)
Comment (kyoungportable) [reply src/Filament/Resources/AuthorResource.php:36]: Yep.  I've addressed the unnecessary variable issue, but I'm creating a separate branch to deal with the linting.

**Comment by jeremy-portable** [comment]
Looks good to me.

**Pull request merged**